features/account/accesibility.feature

```diff
@@ -0,0 +1,21 @@
+Feature: User settings page
+    As a logged user, in any page
+    I want to toggle Use High Contrast UI from The left menu in the account section
+    So I can view page more easily
+    Ticket 008
+
+    Scenario: Activate high contrast ui
+        Given the user is logged in
+        And the user is on "dashboard" page
+        And the user open the "account" side menu
+        When the user click toggle "true" the "high contrast" check
+        And the page is refreshed
+        Then the page should be displayed in high contrast
```

Comment:
```
Given the option of changing the contrast is available through the "account" side menu
When the "high contrast" option is set to "true"
Then the website is displayed in high contrast
```
The when on this sentence can be different but I don't know the implementation, another option could be the following if it makes more sense based on the current implementation
```
When the "high contrast" option is enabled
```
This way, we have a very flexible Gherkin where technically every option in whatever section in the side menu could be manipulated and validated, only the `Then` would be different